python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/platform.j2

```diff
@@ -6,6 +6,11 @@
 {# eos - platform #}
 {% if platform is arista.avd.defined %}
 !
+{%     if platform.trident.l3 is arista.avd.defined %}
+{%         if platform.trident.l3.routing_mac_address_per_vlan is arista.avd.defined(true) %}
+platform trident l3 routing mac-address per-vlan
+{%         endif %}
+{%     endif %}
```

Comment:
```suggestion
{%     if platform.trident.l3.routing_mac_address_per_vlan is arista.avd.defined(true) %}
platform trident l3 routing mac-address per-vlan
{%     endif %}
```

Reply:
Hi Claus, 
I was setting it like that on purpose, cause there are other options under trident.l3 on eos.. I thought in case sb else needs to add more keys in the future. Changing is also fine. Just let me know what you think..

Reply:
Ordering in CLI has been tested:

```
config)#sh run | b platform
platform trident l3 routing mac-address per-vlan
platform trident forwarding-table partition 4
!
```